cloudify_marketplace/provision/bootstrap_simple

```diff
@@ -71,13 +61,9 @@ function bootstrap
     echo "bootstrap failed!"
     exit 1
   fi
```

Comment:
re-add `set -e` when exiting the func

Reply:
and also the func used to `profiles use` at the success case, but now doesn't; the install does it itself, right?

Reply:
Yes, `cfy_manager install/configure` do this.
I'll re add `set -e`